### ISSUE TITLE
Using other receiver front ends with a tayloe mixer

### ISSUE THREAD

**Issue by RecepAGulec** [dawsonjon/PicoRX#76]
Can I and Q outputs of any receiver frontend using a Tayloe mixer be applied to the RSP pico for which you have prepared the software? For example, If the LO_I and LO_Q signals are applied directly to the FST3253, can the I and Q signals produced by the receiver module produced by QRP-Labs be applied to the PICO? (Removing the 74ACT74 integrated circuit)
( https://www.qrp-labs.com/receiver.html )

**Comment by dawsonjon** [comment]
Yes, that's exactly how I built the first prototype! You can just about see it in [this video](https://github.com/dawsonjon/PicoRX/assets/717535/4846c78d-21c3-42e8-bcdf-278cc2bdadcb). You just need to bias the I and Q signals so that they sit in the centre of the ADC supply rail (about 1.5v).
